libsolidity/interface/StandardCompiler.cpp

```diff
@@ -91,6 +92,21 @@ Json::Value formatErrorWithException(
 	return formatError(_warning, _type, _component, message, formattedMessage, location);
 }
 
+/// @_hash contains a Keccak256 hash with the 0x prefix
```

Comment:
Should be `@param _hash contains...` and I think the main description should be in the first line. I usually just mention the params in the description to avoid redundancy (redundancy often makes you take longer to grasp what it does)
```
Returns true iff @a _hash (hex with 0x prefix) is the Keccak256 hash of the binary data in @a _content.
```
Is the `0x` prefix required?